src/types.js

```diff
@@ -1,6 +1,96 @@
 // @flow
 
 /**
+ * Source File Location
```

Comment:
we tried doing something like this:

```js
import type { Breakpoint, Source } from "devtools-client-adapters/types";
export type { Breakpoint, Source };
```

This worked to the extent that we didn't have to copy the types and the debugger would mention when a type was missing, but it didn't help us w/r/t to validating the types.